hoot-js/src/main/cpp/hoot/js/conflate/merging/HighwaySnapMergerJs.cpp

```diff
@@ -112,8 +112,10 @@ void HighwaySnapMergerJs::apply(const FunctionCallbackInfo<Value>& args)
   MergerBase::PairsSet pairs = toCpp<MergerBase::PairsSet>(args[2]);
   vector<pair<ElementId, ElementId>> replaced =
       toCpp<vector<pair<ElementId, ElementId>>>(args[3]);
+  const QString matchedBy = toCpp<QString>(args[4]);
 
   HighwaySnapMergerPtr snapMerger(new HighwaySnapMerger(pairs, sublineMatcher));
+  snapMerger->setMatchedBy(matchedBy);
```

Comment:
All the js conflate scripts conflating linear features actually use `HighwaySnapMerger` for line merger...which seems to be fine. So setting this tag here makes it so it doesn't have to be set in each one of those scripts (like it is for the scripts supporting the other geometry types).